repos/keg-components/src/theme/components/button/contained.js

```diff
@@ -1,45 +1,52 @@
-import { colors } from '../../colors'
 import { transition } from '../../transition'
-import { get } from '@keg-hub/jsutils'
 import { buildTheme } from '../../../utils/styles'
 import { getThemeDefaults } from '../../themeDefaults'
+import { deepMerge, get, noOpObj, checkCall } from '@keg-hub/jsutils'
 
-const defaults = getThemeDefaults()
-const containedStyles = (state, colorType) => {
-  const opacity = get(defaults, `states.types.${state}.opacity`)
-  const shade = get(defaults, `states.types.${state}.shade`)
-  const activeColor = get(colors, `surface.${colorType}.colors.${shade}`)
-  return {
-    main: {
-      $all: {
-        borderWidth: 0,
-        borderRadius: 4,
-        backgroundColor: activeColor,
-        padding: 9,
-        minHeight: 35,
-        opacity,
-      },
-      $web: {
-        cursor: state === 'disabled' ? 'not-allowed' : 'pointer',
-        boxShadow: 'none',
-        ...transition([ 'backgroundColor', 'borderColor' ], 0.5),
-      },
-      $native: {},
-    },
-    content: {
-      color:
-        state === 'disabled'
-          ? get(colors, 'opacity._50')
-          : get(colors, 'palette.white01'),
-        fontSize: 14,
-        fontWeight: '500',
-        letterSpacing: 0.5,
-        textAlign: 'center',
+export const containedInit = (config=noOpObj) => {
+  const { colors, states } = getThemeDefaults()
+  const __transition = transition(config)
+
+  const containedStyles = (state, colorType) => {
```

Comment:
not related to your changes here since im noticing it on develop as well, not sure when this bug was introduced, but for `button` components:
- there's a bug where if you click+drag on a button, it removes some of the styling, until you hover on it again
![buttons](https://user-images.githubusercontent.com/3317835/99315062-4912d400-281f-11eb-828f-6263c6e9e43d.gif)

- for comparison, I looked at the old [keg-components gh page](https://simpleviewinc.github.io/keg-components/?path=/story/button-outline--standard) and it's not doing it there

I can spin off a ticket for this

Reply:
ticket: https://jira.simpleviewtools.com/browse/ZEN-462

Reply:
Hmm, this is interesting. I did a little research on it.
Looks to be related to the latest update to [react-native-web 14](https://github.com/necolas/react-native-web/issues/1719)
  * There were a few changes, that were "fixed", but really didn't fix things for us
    * Potentially related Issues
      * https://github.com/necolas/react-native-web/issues/1264
      * https://github.com/necolas/react-native-web/issues/1604
      * https://github.com/necolas/react-native-web/issues/1640

Reply:
added those links to the ticket